api_core/google/api_core/bidi.py

```diff
@@ -172,7 +172,9 @@ def __init__(self, access_limit, time_window):
 
         self._time_window = time_window
         self._access_limit = access_limit
-        self._past_entries = collections.deque(maxlen=access_limit)  # least recent first
+        self._past_entries = collections.deque(
+            maxlen=access_limit
```

Comment:
unrelated?

Reply:
It was modified during the `black` session.